addons/mrp/models/mrp_production.py

```diff
@@ -1782,6 +1816,9 @@ def action_see_move_scrap(self):
         action['context'] = dict(self._context, default_origin=self.name)
         return action
 
+    def action_view_reception_report(self):
+        return self.env["ir.actions.actions"]._for_xml_id("stock.stock_reception_action")
+
```

Comment:
wouldn't it better to do another xml with a default context {default_production_ids: active_id(s)} ?